### DISCUSSION TITLE
CryptoSmite and "do a dangerous hardware modification."

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1045] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I am currently thinking about unenrolling but a bit stumped on how. As far as I know, I think my Kernver (or whatever) is 3 and I am unable to downgrade. After seeing this I began looking into the dangerous hardware modification also located on the exploit page. This exploit known as the pencil method or the tsunami method (it goes by many names) has only instructions for sh1mmer on the link that it redirects to. I am wondering if anyone can give detailed instructions on how to do this exploit with the other newest exploit (CryptoSmite) instead of the sh1mmer instructions given. (or even i just want to know if those are the only working instructions for anything related to the tsunami exploit) Thanks! By the way, I'm on version: 120.0.6099.235 (Official Build) (64-bit) with a craaskbowl Customation ID and a Nissa board.

**Comment by nathblade16** [comment, marked as answer]
> I am currently thinking about unenrolling but a bit stumped on how. As far as I know, I think my Kernver (or whatever) is 3 and I am unable to downgrade. After seeing this I began looking into the dangerous hardware modification also located on the exploit page. This exploit known as the pencil method or the tsunami method (it goes by many names) has only instructions for sh1mmer on the link that it redirects to. I am wondering if anyone can give detailed instructions on how to do this exploit with the other newest exploit (CryptoSmite) instead of the sh1mmer instructions given. (or even i just want to know if those are the only working instructions for anything related to the tsunami exploit) Thanks! By the way, I'm on version: 120.0.6099.235 (Official Build) (64-bit) with a craaskbowl Customation ID and a Nissa board.

unless you do a hardware mod(which is not worth it) or your kernel version is less than 3, you can't uneroll

btw unerolling is not worth it, you will get into major trouble if you get IT support or hand in your chromebook, plus your admin will know 

otherwise(and you are ignoring all warnings and are doing this under your own liability and risk I take no responsability for your actions) use sh1mmer if your on kernel version 1 or 0 and cryptosmite if your on kernel version 2

**Reply by AnOrangeCone** [reply]
when you need to hand it in reenroll it and if you need IT support you fucked up fix it yourself 90% of they time its as easy as esc power refresh and if you broke the screen you can still reenroll using cryptosmite usb and if needed a external monitor

**Reply by nathblade16** [reply]
> when you need to hand it in reenroll it and if you need IT support you fucked up fix it yourself 90% of they time its as easy as esc power refresh and if you broke the screen you can still reenroll using cryptosmite usb and if needed a external monitor

my chromebook got fried because I charged it using the official charger

**Comment by AnOrangeCone** [comment]
on v119 which i think v3 can downgrade to you can use cryptosmite but not on 120

**Reply by nathblade16** [reply]
no, you won't have kernel version 3 on 119
